# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: f29451ea18dc
# Full Hash: f29451ea18dcf20e024c15c82bb814f9c2d9389e
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - Create DOM specific Module Loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137298
# ==============================================================================

diff -r c0bd1d29be62 -r f29451ea18dc dom/script/moz.build
--- a/dom/script/moz.build	Thu Feb 17 15:57:50 2022 +0000
+++ b/dom/script/moz.build	Thu Feb 17 15:57:50 2022 +0000
@@ -20,7 +20,7 @@
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
     "LoadedScript.h",
-    "ModuleLoader.h",
+    "ModuleLoaderBase.h",
     "ModuleMapKey.h",
     "ScriptDecoding.h",
     "ScriptElement.h",
@@ -33,7 +33,7 @@
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
     "LoadedScript.cpp",
-    "ModuleLoader.cpp",
+    "ModuleLoaderBase.cpp",
     "ModuleLoadRequest.cpp",
     "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
